arduino/smartcar/smartcar.ino

```diff
@@ -85,20 +85,33 @@ void loop() {
     const auto currentTime = millis();
     #ifdef __SMCE__
         static auto previousFrame = 0UL;
-        if (currentTime - previousFrame >= 65) {
+        if (currentTime - previousFrame >= oneSecond) {
           previousFrame = currentTime;
+		  		  
+		  // Publish Camera data
           Camera.readFrame(frameBuffer.data());
-          mqtt.publish("/smartcar/camera", frameBuffer.data(), frameBuffer.size(),
-                       false, 0);
+          mqtt.publish("/smartcar/camera", frameBuffer.data(), frameBuffer.size(),false, 0);
+		  const auto totalDistance = String(travelledDistance());
+		  mqtt.publish("/smartcar/odometer", totalDistance);
+		  const auto distance = String(measureDistance());
+		  mqtt.publish("/smartcar/ultrasound/front", distance);
```

Comment:
Please indent the code appropriately. You can easily do this by pressing "Ctrl + T" on the Arduino IDE.